### PR TITLE
feat: genai-demo

### PR DESCRIPTION
**Pull request by tgpadua**
*Issue #, if available:*

*Description of changes:* adding new demo app

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: genai-hitl-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+5. Generates a custom avatar avatar as a S3 pre-signed url
+
+Solution decomposes generation and selection of title, description and avatar to simpler prompts by first generating the title and description. This showcases how prompt chaining can be used with Step Functions.
+
+![app](app.png)
+
+### Deploy
+Make sure your account is [enabled to access](https://console.aws.amazon.com/bedrock/home?#/modelaccess) Amazon Bedrock models anthropic-claude-v2 and anthropic.claude-v2:1
+
+1. Deploy the stack
```
[the remark this answers is not in the document]
Comment (tgpadua) [reply genai-hitl-demo/README.md:20]: Yes, we do. Thanks, I was missing that step.

File: genai-hitl-demo/app.png
[the remark this answers is not in the document]
Comment (tgpadua) [reply genai-hitl-demo/app.png:1]: There is no open source image being used in the solution. If you're referring to the screenshot, that was generated by Amazon Bedrock.

**Comment by tgpadua** [comment]
Thanks for the feedback. All the suggestions were incorporated in the README file.

**Review by youngjeong46 — APPROVED** [review]
LGTM.

**Comment by youngjeong46** [comment]
LGTM

**Pull request merged**